transports/onion/tests/mod.rs

```diff
@@ -0,0 +1,88 @@
+use std::error::Error;
+
+use libp2p::{
+    dns, identity, mplex, noise,
+    ping::{self, Behaviour},
+    swarm::SwarmEvent,
+    tcp, yamux, PeerId, Swarm, Transport,
+};
+use libp2p_onion::OnionClient;
+
+#[cfg(feature = "async-std")]
```

Comment:
Instead of this, you can define the test in `Cargo.toml` explicitly and say it requires certain features. See here for example: https://github.com/Restioson/xtra/blob/b538a5a41130fbfeb3e14eb816e1010f9167d17a/Cargo.toml#L114-L116

Reply:
Resolved in latest commit.